### PR TITLE
docs: redo Color page

### PR DESCRIPTION
**Pull request by marionnegp**
## What I did

1.

## Testing Instructions

1. Check [DP](https://deploy-preview-1321--red-hat-design-system.netlify.app/foundations/color/)

## Notes to Reviewers

@coreyvickery, some changes while you were out:
- On the "Theming" subpage, I changed "Custom colors" to "Custom theme" because Ryan said Brand will reject requests for new colors. I figured custom theme is what you meant.
- I updated the content under "Usage" > "Surface". The background of a button isn't really considered a surface, so the paragraph had to be reworded a bit.
- I also replaced "component" with "element and pattern" wherever it appear, except for the "Graphical objects and UI components" section.

### CODE REVIEW

**Comment by marionnegp** [comment]
@coreyvickery, do you know what "contact the Brand team" should link to?

@zeroedin or @bennypowers, can `<rh-tag>` be added to the template? I tried adding it to `layout-foundations.njk` frontmatter and using `{% tag %}`, but that caused an error. Also, does it matter whether I use HTML or markdown to create tables? Both are used on UX dot.

**Comment by coreyvickery** [comment]
@marionnegp Not sure, Ryan or Libby would know.

**Comment by coreyvickery** [comment]
@marionnegp Where did the information in the second bullet come from?

**Comment by coreyvickery** [comment]
@marionnegp Looks great, some small edits.

### Overview
- [ ] Under the `Primitive and semantic colors` heading, bold the word `primitives`

### Theming
- [ ] The light theme image looks a bit blurry when compared to the dark theme image. If there's no fix, don't worry about it

### Usage
- [ ] Under the `Best practices` heading under `Custom themes`, only the words `Design system` should be linked

### Palette
- [ ] Ensure all columns in all tables are the same width

**Review by bennypowers — APPROVED** [review]
I updated the markup and some of the content (e.g. phrasing content between headers)

I think the color palette tables should be *removed* and replaced with token tables, or better yet links to the token pages. The palette page should contain directions on how to combine token colours in design projects.

**Comment by bennypowers** [comment]
@nikkimk if you have time can you go over the a11y markup here? can we replace the refs with shared reference links from an include? am i being too precious about that?

**Comment by bennypowers** [comment]
I think the theming content should move to #1298

**Comment by bennypowers** [comment]
@marionnegp if you like i can merge #1298 into this PR and we'll continue here

**Comment by coreyvickery** [comment]
> @marionnegp if you like i can merge #1298 into this PR and we'll continue here

@bennypowers I would prefer the theme stuff stay in #1298 and the PRs stay separate. I guess we can remove any mention of theming from the Color pages until the new Theming section is done.

**Comment by bennypowers** [comment]
thats fine, but the theming content here is *good*, and #1298 has lots of stub pages without content, sooo....

File: docs/foundations/color/theming.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            state="warning" %}
+Do not apply the Red Hat red color to text in dark environments unless
+it meets [WCAG 2.1 AA][wcag-2-1-aa] requirements.
+{% endalert %}
+
+{% example palette="darkest",
+          alt="examples of several elements against a dark gray surface",
+          src="/assets/color/dark-theme.png" %}
+
+### Desaturated theme {% tag color="purple" %}Beta{%endtag%}
```
Comment (bennypowers) [inline docs/foundations/color/theming.md:52]: the word `desaturated` is new and doesn't match with the code, except in the case of tile.

is desaturated a color palette? is it a variant on any of the existing color palettes?

**Comment by markcaron** [comment]
I'd like to see a few changes in the way we present images in docs, especially when it's a `do this` / `don't do this` image.

1. Separate out the `do this` / `don't do this` examples into separate images, rather than combining them into a single image. When combined, it's difficult to write `alt` text as well as provide caption/supporting text.
2. Make the green check and red alert icons a separate `tag` and pair it with HTML text, like we do on the Brand site. See image below.
3. Provide caption and supporting text with every image. This helps provide localized context for each example. If we need to, we can use `rh-card` or a simple column/grid structure. See image below.

image

Second, I think we could use a lot less vertical space and create more interesting rhythm through these docs if we create the occasional column structures. One place in particular where we present the color palette tokens. Using a 100% width table creates a lot of white space between the swatch, the token and the hex code—which is a bit of an accessibility issue since the columns can seem disassociated (due to the white space). I think we can present the color tables as 50% width columns (on medium / large screens), so we get a closer pairing of text and swatch as well as a pleasant visual of color gradients side-by-side.

For example:
image

**Review by coreyvickery — APPROVED** [review]

**Pull request merged**